_docs-2/troubleshooting/tools.md

```diff
@@ -59,6 +59,41 @@ Likewise, `rfile-info` will dump the key-value pairs and show you the contents o
     row columnFamily:columnQualifier [visibility] timestamp deleteFlag -> Value
     ...
 
+### Encrypted Files
+
+To examine an encrypted rfile the necessary encryption properties must be provided to the utility. To do this
+the `accumulo.properties` file can be copied, the necessary encryption parameters added, and then the properties file can 
+be passed to the utility with the `-p` argument. 
+
+For example, if using `PerTableCryptoFactory` and the `AESCryptoService`, you would need the following properties in
+your accumulo.properties file:
+
+    general.custom.crypto.key.uri=<path-to-key>/data-encryption.key
+    instance.crypto.opts.factory=org.apache.accumulo.core.spi.crypto.PerTableCryptoServiceFactory
+    table.crypto.opts.service=org.apache.accumulo.core.spi.crypto.AESCryptoService
+
+Example output:
+
+    $ accumulo rfile-info /accumulo/tables/1/default_tablet/A0000004.rf -p <path-to-properties>/accumulo.properties
```

Comment:
The example should use the fully qualified URI, rather than just the path, in order to avoid the subsequent warning, and set a better example to users. Something like `hdfs://localhost:9020/accumulo/tables...`